example/windows/ProgressViewExample/ProgressViewExample.vcxproj

```diff
@@ -0,0 +1,182 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="16.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('..\packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.props')" />
```

Comment:
nit, use a newer version of cppwinrt

Reply:
Our native modules setup for v62 mentions this version - shouldn't we follow our own guidelines in this case?
(Asking, since I'm now working on some native modules , too)

Reply:
@rectified95 we should arguably be updating the docs. there are new versions of cppwinrt being released every month, it's usually a good practice to keep up to date with updates. Do we mention a specific version in our docs? maybe we need to add a comment there to say "use the newest version available".

Reply:
The docs say to use this specific version "_Right click on the project and choose Manage Nuget Packages...
Select version 2.0.200316.3 for Microsoft.Windows.CppWinRT package". I'm honestly not sure why but @kaiguo might know, I can also just update it to the newest version and see if it breaks the build or not

Reply:
We used to get compile errors with newer versions of cpp/winrt so Di added the version requirements in the doc. But that was back in 0.61 era, maybe it's no longer required. Try updating to the latest and see if it works?

Reply:
note that cppwinrt does get bug fixes, e.g. there was a bug in an earlier version of cppwinrt that we worked around by adding an addref to the app instance (where it would normally be leaking the instance)

Reply:
I updated it and it didn't break the build, may need to update docs then if other modules get the same outcome.